src/library/coverartcache.cpp

```diff
@@ -13,9 +13,9 @@ namespace {
 
 mixxx::Logger kLogger("CoverArtCache");
 
-QString pixmapCacheKey(quint16 hash, int width) {
+QString pixmapCacheKey(quint16 hash, int actualWidth) {
     return QString("CoverArtCache_%1_%2")
-            .arg(QString::number(hash)).arg(width);
+            .arg(QString::number(hash)).arg(actualWidth);
```

Comment:
I'll change it, but what is the problem with .arg().arg()?

Reply:
It parses the format string twice. First it creates arg() creates new intermediate string with only %2. 

The multi arg version reserves first the whole destination buffer and then copies the result into it. 
https://github.com/qt/qtbase/blob/53d62b8fcbb639bd625777c8f1c01764445fb1c4/src/corelib/tools/qstring.cpp#L9010

It could be interesting to compare this with the + operator.

Reply:
We should keep am eye on using % instead of + when concanating string this should be the fastest method.